addons/payment_adyen/models/payment_transaction.py

```diff
@@ -94,12 +94,20 @@ def _send_payment_request(self):
             data.update(captureDelayHours=0)
 
         # Make the payment request to Adyen
-        response_content = self.provider_id._adyen_make_request(
-            url_field_name='adyen_checkout_api_url',
-            endpoint='/payments',
-            payload=data,
-            method='POST',
-        )
+        try:
+            response_content = self.provider_id._adyen_make_request(
+                url_field_name='adyen_checkout_api_url',
+                endpoint='/payments',
+                payload=data,
+                method='POST',
+            )
+        # Log the error message on linked documents' chatter
+        except ValidationError as e:
+            if self.operation == 'offline':
+                self._set_error(e)
```

Comment:
It'd make more sense to move that comment on the line that actually logs the error message.
```suggestion
        except ValidationError as e:
            if self.operation == 'offline':
                self._set_error(e)  # Log the error message on linked documents' chatter.
```